lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2116,6 +2116,16 @@ namespace {
           continue;
         }
 
+        if (auto friendDecl = dyn_cast<clang::FriendDecl>(m)) {
+          if (auto d = VisitFriendDecl(friendDecl)) {
```

Comment:
This should be done automatically on line 2147:
```
Decl *member = Impl.importDecl(nd, getActiveSwiftVersion());
```

Is there some reason that doesn't work?

Reply:
Could be! Testing.

Reply:
`FriendDecl` is not a `NamedDecl` so it does not get to that line.

Reply:
What if you did this instead?
```suggestion
          if (auto d = VisitFriendDecl(friendDecl)) {
            m = friendDecl->getFriend();
```

Reply:
What about my above suggestion?

Reply:
Should work, would have to add to `result->addMember` later on. But offline sounds like maybe we should make this global